libraries/reliable_amqp_publisher/include/eosio/reliable_amqp_publisher/reliable_amqp_publisher.hpp

```diff
@@ -0,0 +1,52 @@
+#include <string>
+#include <memory>
+#include <vector>
+#include <functional>
+
+#include <fc/io/raw.hpp>
+
+namespace eosio {
+
+/*
+ * A reliable_amqp_publisher manages a connection to a single AMQP server that publishes messages to an exchange
+ * in a reliable manner: connections to the AMQP server are retried, publishing occurs in a transaction,
+ * and unconfirmed messages are kept in a queue that is persisted to disk on destruction.
+ *
+ * Be aware that the reliable_amqp_publisher will NOT create the exchange. It requires the exchange
+ * to already exist. Publishing to a non-existing exchange will effectively block
+ * reliable_amqp_publisher's queue until the exchange exists because the queue will be unable to make progress.
+ *
+ * At a large enough uncomfirmed queue depth (currently on the order of 1 million entries) reliable_amqp_publisher
+ * will start dropping messages after logging an error.
+ */
+
+class reliable_amqp_publisher {
+   public:
+      /// Create a reliable queue to the given server. "name" is used as filename for persisting messages
+      ///  so should be unique to usage
```

Comment:
I either don't understand or this comment is not correct. `unconfirmed_path` unmolested is used for the file name. Not sure what "name" refers to.

Reply:
thanks you are right this comment didn't get changed after some refactoring